src/libstd/panicking.rs

```diff
@@ -221,9 +221,9 @@ impl<'a> PanicInfo<'a> {
     ///
     /// panic::set_hook(Box::new(|panic_info| {
     ///     if let Some(location) = panic_info.location() {
-    ///         println!("panic occured in file '{}' at line {}", location.file(), location.line());
```

Comment:
Now there's a new tidy error:

```
tidy error: /checkout/src/libstd/panicking.rs:224: trailing whitespace
```

You're going to need to remove that space at the end of this line